test/docker/run_docker.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # reset test harness
 rm -rf test-harness
 rm -rf test/features
-git clone --single-branch --branch feature/box-storage https://github.com/algorand/algorand-sdk-testing.git test-harness
+git clone --single-branch --branch use_latest https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
https://github.com/algorand/go-algorand/pull/4154 is merged.  Safe to revert branch change.

```suggestion
git clone --single-branch --branch feature/box-storage https://github.com/algorand/algorand-sdk-testing.git test-harness
```